core/arch/arm/include/mm/core_mmu.h

```diff
@@ -86,6 +86,7 @@
  * MEM_AREA_IO_SEC:   Secure HW mapped registers
  * MEM_AREA_RES_VASPACE: Reserved virtual memory space
  * MEM_AREA_TA_VASPACE: TA va space, only used with phys_to_virt()
+ * MEM_AREA_TEE_RAM_UNCACHED:  Same as MEM_AREA_TEE_RAM, but cache disabled.
```

Comment:
Ok. The strange thing is that these changes add code that is not used yet.
When doing so, it is highly probable that such code is not maintained and ends in adding questions rather providing some support.

Maybe we should split this PR in 3 pieces. It could probably help their integration.